setup.py

```diff
@@ -51,9 +51,14 @@ def read_requirements(filename):
     "boto3>=1.26.131,<2.0",
     "cloudpickle==2.2.1",
     "google-pasta",
+    "google-api-core==2.8.0",
+    "google-auth==2.6.6",
+    "google-cloud-core==2.3.0",
+    "google-cloud-storage==2.3.0",
```

Comment:
@knikure lets discuss on the reason for pulling in so many direct dependencies, we have to check on the size concerns before merging this in.